services/autoscaling/src/simcore_service_autoscaling/core/application.py

```diff
@@ -49,10 +51,14 @@ def create_app(settings: ApplicationSettings) -> FastAPI:
 
     # EVENTS
     async def _on_startup() -> None:
```

Comment:
Same as above: we always show banner upon startup and shtuone in a particular order. ANy service in _meta module could configura the base class and we could get all this functionality very similar

Every iteration i try to remove more and more of this code so we can create new services from scratch faster

Reply:
good point.